MAUI/PDF-Viewer/Form-Filling.md

```diff
@@ -23,6 +23,8 @@ You can load and fill in the following form fields in a PDF document using the P
 * List box
 * Button
 
+N> The selectable highlight color will be applied to a form field only if its background color is transparent. Otherwise, the specified background color will be displayed in the form field.
```

Comment:
The content is not clear and the version is not mentioned. Please get review from development team and bring again.